### PR TITLE
Issue 3827 - Text being cut off

### LINKED ISSUE #3827: Text being cutoff (missing ellipsis)
### Tasks
- [x]  Fix group names truncated but can overflow when nested
image
- [x] Edit profile modal 
image
- [x] dashboard list item (includes edit federation)
image
- [x] Expanded container list item
image
- [x] Breadcrumbs trims only the last segment, which might be fine, but this means that the other segments break the line
image
> Also, the overflowing may look pretty bad if that happens 
image

- [x] Safetibase editable title 
image
- [x] Compare Card 
image
- [ ] <del>Sequences (although we haven't really tackled this card yet)</del>
image
- [x] Ticket title)
image
- [x] Latest Revision Title
image

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #3827 

#### Description
Fixed text overflow in a few places
P.S. look how cool the breadcrumbs behaves like now!!

### CODE REVIEW

File: frontend/src/v5/ui/v4Adapter/overrides/cards/compare.overrides.ts [hunk 1]
```diff
@@ -35,4 +35,10 @@ export default css`
 	${FilterPanel} .MuiInputBase-root {
 		padding-left: 39px;
 	}
+
+	${Name} {
+		max-width: 317px;
+		overflow: hidden;
+		text-overflow: ellipsis;
```
Comment (The-Daniel) [inline frontend/src/v5/ui/v4Adapter/overrides/cards/compare.overrides.ts:42]: I think this needs a `white-space: nowrap;`
image

**Review by The-Daniel — CHANGES_REQUESTED** [review]

**Comment by The-Daniel** [comment]
image
It's possible to make the author name wrap onto multiple lines and flow outside its containers

**Comment by Amantini1997** [comment]
> image Latest revision in the uploads model destination autocomplete might need truncation

> image
Ticket names don't truncate

I added both of these to the main issue list

File: frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemRow/dashboardListItemRow.styles.ts [hunk 2]
```diff
@@ -37,14 +37,13 @@ export const Container = styled.div<{ selected?: boolean }>`
 			border: none;
 		}
 		
-		${TextOverflowStyles.Container} {
-			&:after {
-				${TextOverflowStyles.fadeToLeft(theme.palette.secondary.main)}
-			}
+		${TextOverflowContainer}::after {
+			border: solid 2px red;
```
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemRow/dashboardListItemRow.styles.ts:41]: Ah yes, indeed

File: frontend/src/v5/ui/routes/viewer/groups/groupItem/groupItem.styles.ts [hunk 3]
```diff
@@ -20,31 +20,50 @@ import { StyledIconButton } from '@/v4/routes/teamspaces/components/tooltipButto
...
+
+	~ ${Separator} {
+		margin: 0;
+		border-bottom-width: 1px;
+	}
 `;
 
 const IconSize = css`
-	width: 30px;
+	min-width: 30px;
```
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/routes/viewer/groups/groupItem/groupItem.styles.ts:66]: @mosment3DRepo

File: frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemTitle/dashboardListItemTitle.styles.ts [hunk 4]
```diff
@@ -27,17 +27,18 @@ export const Title = styled(Button).attrs({
...
 	${({ theme, selected }) => selected && css`
 		color: ${theme.palette.primary.contrast};
 	`}
 
-	& span {
+	a {
 		text-overflow: ellipsis;
-		white-space: nowrap;
 		overflow: hidden;
+		max-width: 100%;
```
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemTitle/dashboardListItemTitle.styles.ts:40]: Let's wait to see what @sanmont3drepo says

File: frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemTitle/uploadListItemTitle.styles.ts [hunk 5]
```diff
@@ -22,8 +22,12 @@ export const DashboardListItemTitle = styled(ListItemTitle)`
...
 	padding-right: 5px;
 	box-sizing: border-box;
+
 	.MuiButton-root {
 		cursor: default;
 		text-decoration-line: none;
+		text-overflow: ellipsis;
+		overflow: hidden;
+		display: inline-block;
 	}
```
Comment (The-Daniel) [inline frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemTitle/uploadListItemTitle.styles.ts:32]: image
The latest revision is still not truncated (In the destination input of the upload file modal)

**Review by The-Daniel — CHANGES_REQUESTED** [review]

**Review by The-Daniel — DISMISSED** [review]

File: frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemTitle/dashboardListItemTitle.styles.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (sanmont3drepo) [reply frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListItem/components/dashboardListItemTitle/dashboardListItemTitle.styles.ts:40]: > I remember it being discussed whether text overflow css should be moved to the theme. Or perhaps we should be making use of the TextOverflow control?

Lets keep it for now this way.

**Comment by sanmont3drepo** [comment]
In the viewer the breadcrumbs are missing the ellipsis
image

**Review by sanmont3drepo — APPROVED** [review]

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**